app/mailers/account_mailer.rb

```diff
@@ -8,12 +8,14 @@ def confirmation_instructions(record, token, opts = {})
   end
 
   def reset_password_instructions(record, token, opts = {})
-    EventAPI.notify('system.notification.account', reset_password_token: token)
+    EventAPI.notify('system.notification.account',
```

Comment:
I think the naming keys here are the wrong way round, it should actually be `system.account.notification` right?

Also - how comes this isn't actually `system.account.forgotten_password`?